FIAT/lagrange.py

```diff
@@ -6,6 +6,65 @@
 # SPDX-License-Identifier:    LGPL-3.0-or-later
 
 from FIAT import finite_element, polynomial_set, dual_set, functional
+from FIAT.finite_element import make_barycentric_lattice_coordinates, make_index_permutations
+import numpy as np
+
+
+def make_entity_permutations(dim, npoints):
+    r"""Make orientation-permutation map for the given
+    simplex dimension, dim, and the number of points along
+    each axis
+
+    As an example, we first compute the orientation of a
+    triangular cell:
+
+       +                    +
+       | \                  | \
+       2   0               47   42
+       |     \              |     \
+       +--1---+             +--43--+
+    FIAT canonical     Mapped example physical cell
+
+    Suppose that the cone of the physical cell is given by:
```

Comment:
cone isn't a word in the FIAT vocabulary.

Reply:
Fixed. Thanks.